OMEdit/OMEditLIB/OMS/ElementPropertiesDialog.cpp

```diff
@@ -439,57 +439,47 @@ bool ElementPropertiesDialog::eventFilter(QObject *pObject, QEvent *pEvent)
  * helper function to restore default start values read from modeldescription.xml for fmus
  * and 0 for other systems
  */
-void ElementPropertiesDialog::restoreDefaultStartValue(QString pName, QLineEdit * pLineEdit)
+void ElementPropertiesDialog::deleteStartValueAndRestoreDefault(QString pName, QLineEdit * pLineEdit)
```

Comment:
`pName` should be `name`. Actually more better is `const QString name`.

Reply:
fixed with https://github.com/OpenModelica/OpenModelica/pull/6857/commits/8738cbb0007a43976ed0b1f1007d672c75c04917